source/manuals/programming-languages/nodejs.html.md.erb

```diff
@@ -0,0 +1,118 @@
+---
+title: Node.js styleguide
+---
+
+# <%= current_page.data.title %>
+
+For JavaScript at GDS, we use the [standard] styleguide, and that includes for Node.js.
+This gives us consistency across frontend and backend JavaScript implementations, and enables developers to code against the same set of rules wherever their JS is running.
+
+We generally do not use Node.js at GDS for any ‘significant application logic’, however that does not mean we don't use it.
+GOV.UK Pay uses Node to create thin presentation layers that retrieve data from APIs, but do not themselves store data.
+
+However even in these examples, Node applications can include a sizable amount of code.
+So there are additional considerations to be taken into account in order to keep the code readable and maximise maintainability.
+
+## ES2015 and beyond
+One of the best parts of coding Node is that you can use most of the latest JavaScript features.
+This can often (but not always) make for smaller, more easily readable and maintainable code,
+though judgement should be exercised as to when this is the case.
+
+In Node.js applications, it is better to use `const` and `let` rather than `var`.
+
+A useful guide for what ES2015+ features are supported in each version of Node.js can be found at [Node Green].
+
+## Import statements and strict mode
+
+Each `.js` file in your application should be in [strict mode], `'use strict'` should be at the top of each file,
```

Comment:
If we're using ES6 this is not necessary right? https://stackoverflow.com/questions/31685262/not-recommended-to-use-use-strict-in-es6

Reply:
Node uses commonJS modules rather than ES6 modules, meaning strict mode is still required.

In a node module without strict mode doing
``` foo = 'bar'```
creates a `foo` property on the global object, i.e `global.foo`, same as like the window object in the browser.

Reply:
Also sorry for taking like 2 months to reply, I got a little distracted. I'm on it now.